### DISCUSSION TITLE
Showcase Removal Notice

### DISCUSSION THREAD

**Discussion by JonasKruckenberg** [kimlimjustin/xplorer#230] [General]
Hello friends of Tauri,
With the upcoming v1 release and accompanying overhaul of the https://tauri.studio website, we will be removing the showcase section in favor of [**Awesome Tauri**](https://github.com/tauri-apps/awesome-tauri). 
If you want to keep your project featured, please submit it to that list instead, but please note the rules for submissions have changed.

Thank you for all your support in getting Tauri to this point!

Jonas,
the Tauri team

**Comment by kimlimjustin** [comment]
Thanks for your notice, but there's already Xplorer on the list haha 😆

**Reply by JonasKruckenberg** [reply]
Yes I know, had to notify you nontheless 🤷🏻‍♂️
